ferveo/src/pvss.rs

```diff
@@ -395,7 +401,7 @@ pub(crate) fn aggregate<E: Pairing>(
 
     // So now we're iterating over the PVSS instances, and adding their coefficients and shares, and their sigma
     // sigma is the sum of all the sigma_i, which is the proof of knowledge of the secret polynomial
-    // Aggregating is just adding the corresponding values in pvss instances, so pvss = pvss + pvss_j
+    // Aggregating is just adding the corresponding values in PVSS instances, so PVSS_i = PVSS_(i-1)  PVSS_i
```

Comment:
```suggestion
    // Aggregating is just adding the corresponding values in PVSS instances, so PVSS = PVSS + PVSS_i
```
I guess you still mean that the aggregate is the iterative sum of transcripts?